.gitignore

```diff
@@ -8,3 +8,4 @@ build/
 .bundle/
 ruby/
 .DS_Store
+themes/
```

Comment:
Is this needed?

Reply:
As I mentioned, I have been exploring a port to Hugo, which uses a gitignored `themes` directory. So when I changed back to this branch, it was visible in my working directory and it seemed easy enough to just gitignore it here too. I can delete if you feel strongly about it, but it's not hurting anything.

Reply:
I can accept this change in this pull request but we should not put a change that isn't related to the target topic branch in general. :-)